product_setup_page/src/components/ProductSetup.jsx

```diff
@@ -87,6 +93,7 @@ export default function ProductSetup(props) {
       ))}
       <Button label="Save" appearance="primary" onClick={saveMacros} updateMacroDefinition={updateMacroDefinition} />
       {response && <pre>{response}</pre>}
+      {isLoading ? <div id="spinner"></div>: null}
```

Comment:
Please send me screenshot of loader.

Reply:
![image](https://github.com/CrossRealms/Splunk-Cyences-App-for-Splunk/assets/57986609/c58ac622-bdc8-4df8-8e9d-f10a73f5ceec)